spec/views/people/show.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe "people/show", type: :view do
+  before(:each) do
+    @person = assign(:person, Person.create!(
+      :bio => "MyText",
```

Comment:
Indent the first parameter one step more than `Person.create!(`.<br>Use the new Ruby 1.9 hash syntax.